consensus/spos/consensusMessageValidator_test.go

```diff
@@ -104,7 +106,7 @@ func TestCheckMessageWithSignatureValidity_InvalidSignatureShareSize(t *testing.
 	assert.True(t, errors.Is(err, spos.ErrInvalidSignatureSize))
 }
 
-func TestCheckMessageWithSignatureValidity_ShouldWork(t *testing.T) {
+func TestCheckMessageWithSignatureValidity_SignatureShareShouldWork(t *testing.T) {
```

Comment:
I think that TestCheckMessageWithSignatureShareValidity_ShouldWork, is ok

Reply:
done.